backend/app/models/route.py

```
@@ -2,15 +2,60 @@
+    投稿に関連するルート情報を表すモデル
+    """
+
     __tablename__ = 'routes'
+
     id = db.Column(db.Integer, primary_key=True)
     start_location = db.Column(db.String(255), nullable=False)
     end_location = db.Column(db.String(255), nullable=False)
     distance = db.Column(db.Float, nullable=False)
     duration = db.Column(db.Float, nullable=False)
     post_id = db.Column(db.Integer, db.ForeignKey('posts.id'), nullable=False)
+    difficulty_level = db.Column(db.Integer, nullable=True) # 1. 初級者向け, 2. 中級者向け, 3. 上級者向け
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)